pkg/operator/hive/hive.go

```diff
@@ -333,3 +373,27 @@ func (r *ReconcileHiveConfig) runningOnOpenShift(hLog log.FieldLogger) (bool, er
 
 	return len(list.APIResources) > 0, nil
 }
+
+func computeHiveControllersConfigHash(data map[string]string) string {
+	hasher := md5.New()
+	hasher.Write([]byte(fmt.Sprintf("%v", data)))
+	return hex.EncodeToString(hasher.Sum(nil))
+}
+
+func setHiveControllersConfig(config *hivev1.ControllerConfig, hiveControllersConfigMap *corev1.ConfigMap, controllerName string) {
+	if config.ConcurrentReconciles != nil {
+		hiveControllersConfigMap.Data[fmt.Sprintf("%s-concurrent-reconciles", controllerName)] = strconv.Itoa(int(*config.ConcurrentReconciles))
```

Comment:
Move the formats into constants so that they can be shared by the controllers code when reading the data.